hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -559,7 +559,7 @@ int SaraNcpClient::waitReady() {
         // Disable voltage translator
         modemSetUartState(false);
         // Hard reset the modem
-        modemHardReset();
+        modemHardReset(true);
```

Comment:
What is the optimization here?  I see we now if initialization of the modem fails, we hard reset the modem which results in it turning off, and we don't turn it back on for R410.  Was there a double turn on occurring and now there's only one?  Any chance the turn on will not re-occur?